dtos/user_dto.go

```diff
@@ -13,6 +11,6 @@ type Login struct {
 }
 
 type LoginReturn struct {
```

Comment:
type is user for login return and not string

Reply:
yup. I will fix it